operator/v1/types_console.go

```diff
@@ -132,6 +132,11 @@ type ConsoleCustomization struct {
 	// +kubebuilder:validation:Optional
 	// +optional
 	AddPage AddPage `json:"addPage,omitempty"`
+	// perspectives allows enabling/disabling of perspective(s) that user can see in the Perspective switcher dropdown.
+	// +listType=map
+	// +listMapKey=id
+	// +optional
+	Perspectives []Perspectives `json:"perspectives"`
```

Comment:
nit: the type should be singular imho, the attribute name and json name should be still plural.

```suggestion
	Perspectives []Perspective `json:"perspectives"`
```